include/dev/TPS2HB50BQ1.hpp

```diff
@@ -0,0 +1,77 @@
+// Driver for power switch
+#ifndef LVSS_TPS2HB50BQ1_HPP
+#define LVSS_TPS2HB50BQ1_HPP
+
+#include <EVT/io/ADC.hpp>
+#include <EVT/io/CAN.hpp>
+#include <EVT/io/CANOpenMacros.hpp>
+#include <EVT/io/CANopen.hpp>
+#include <EVT/io/GPIO.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace LVSS {
+
+/**
+ * Power switch for LVSS
+ * /datasheets/TPS2HB35-Q1 Power Switch Data Sheet.pdf
+ */
+class TPS2HB50BQ1 {
+public:
```

Comment:
Forgot to mention that your functions should have doxygen on them

Reply:
You'll need to add this for the constructor and the function below